README.md

```diff
@@ -62,6 +62,8 @@ function NewComponent(props) {
 }
 ```
 
+Some things in the Design System, such as the universal focus styling, are namespaced to the `.nypl-ds` class. To include those in your app, add `.nypl-ds` to whichever wrapper tag makes sense in your application.
```

Comment:
There are some things in the DS that will look awfully funny (`Input` comes to mind) without this declaration, but redeclaring the box model on every component feels very backward... I suppose just so people are aware that visual issues in some consumer apps might arise if people don't add this wrapper class to their application